javascript/triage_bot_using_SDKv3/app.js

```diff
@@ -0,0 +1,87 @@
+// This script uses Asana node 3.0.1
+// npm install asana@3.0.1
+// Seee the SDK repo for mroe infomation:  https://github.com/Asana/node-asana
```

Comment:
Nit: spelling mistake `mroe` -> `more` also extra space between `infomation:` and `https://github.com/Asana/node-asana`